aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -33,6 +33,11 @@ module aptos_token::token {
     // URI lengths: Mean: 76.97, StdDev: 37.41, 95th%: 157, 99th%: 199 (http://www.supermind.org/blog/740/average-length-of-a-url-part-2)
     const MAX_URI_LENGTH: u64 = 512;
 
+    // Property key stored in default_properties controlling who can burn the token.
+    // the corresponding property value is BCS serialized bool.
+    const BURNABLE_BY_CREATOR: vector<u8> = b"BURNABLE_BY_CREATOR";
+    const BURNABLE_BY_OWNER: vector<u8> = b"BURNABLE_BY_OWNER";
+
```

Comment:
can we namespace this? i.e `$__` or something to ensure only we can?

Reply:
these two properties are also supposed to be entered by creator to control the token burn behavior. So this is not limited to us. They can put any property key name they want.